examples/webserver/streaming/src/main/java/io/helidon/examples/webserver/streaming/Main.java

```diff
@@ -25,7 +25,7 @@
  */
 public class Main {
 
-    static final String LARGE_FILE_RESOURCE = "/large-file.bin";
+    static final String LARGE_FILE_PATH = "target/classes/large-file.bin";
```

Comment:
This assumes the current working directory is the example's directory.
How about we don't have a default download file, and in order to exercise the example you must first upload a file ?

We can provide `large-file.bin` as a specimen to be used for the steps. We 'd put the file at the top-level next to `README.md`.

Reply:
Done

Reply:
Done